java/api/src/main/java/org/ray/api/runtime/RayRuntime.java

```diff
@@ -117,40 +140,18 @@
   <T> RayActor<T> createActor(RayFunc actorFactoryFunc, Object[] args,
       ActorCreationOptions options);
 
-  RuntimeContext getRuntimeContext();
-
-  /**
-   * Invoke a remote Python function.
-   *
-   * @param moduleName Module name of the Python function.
-   * @param functionName Name of the Python function.
-   * @param args Arguments of the function.
-   * @param options The options for this call.
-   * @return The result object.
-   */
-  RayObject callPy(String moduleName, String functionName, Object[] args, CallOptions options);
-
-  /**
-   * Invoke a remote Python function on an actor.
-   *
-   * @param pyActor A handle to the actor.
-   * @param functionName Name of the actor method.
-   * @param args Arguments of the function.
-   * @return The result object.
-   */
-  RayObject callPyActor(RayPyActor pyActor, String functionName, Object[] args);
```

Comment:
Please avoid reordering methods. This makes it harder to track the changes in commit history and to solve merge conflicts.

Reply:
Lint says overload functions should be together. So, I reordered the methods.